src/Components/Tile.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import {Square, Text} from '@chakra-ui/react';
+
+
+
+function Tile(props: any) {
+
+    const [color, setColor] = useState<string>("#626887")
```

Comment:
Do you need to put color as a stateful value?

Reply:
This color is for the background of the tiles, so that they get darker when clicked.